Changed it's to it is in all .rst files (#491)
ansible/ansible-documentation: docs/docsite/rst/porting_guides/porting_guide_7.rst

Reviewer: Please don't change the porting_guides. These files are automatically generated.

Author: @gotmax23 I have reverted changes to `porting_guides.rst` , `spelling_word_choice.rst`, `developing_program_flow_modules.rst`. Let me know if I have to change anything else.

Reviewer: Please revert changes to *all* the porting_guides.

Reviewer: > Please revert changes to _all_ the porting_guides.

Has this been done yet?

Author: yes, I have copied the `porting_guides` folder from the main project and replaced it in my repo.

Reviewer 2: Copying over unfortunately isn't the correct approach. This PR now has a batch of porting guide changes based on that copying of files from devel:
![image](https://github.com/ansible/ansible-documentation/assets/30267855/63218f8f-e8c6-4aea-8c62-8a9d4498b09d)

Normally a git rebase would work, but I'm not sure in this instance because I've never done a file copy and then tried a rebase... @gotmax23 what do you think? Maybe the best thing would be a git revert on the last commit?

~~~~~~~~~~~~

- Deprecate ability of lookup plugins to return arbitrary data. Lookup plugins must return lists, failing to do so will be an error in 2.18. (https://github.com/ansible/ansible/issues/77788)

> - Encryption - Deprecate use of the Python crypt module due to it is impending removal from Python 3.13

Reviewer 2: ```suggestion
- Encryption - Deprecate use of the Python crypt module due to its impending removal from Python 3.13
```
